vmm/src/vmm_config/logger.rs

```diff
@@ -41,6 +41,7 @@ pub struct LoggerConfig {
     #[serde(default)]
     pub show_log_origin: bool,
     /// Additional logging options.
+    #[cfg(target_arch = "x86_64")]
```

Comment:
I'd rather not remove it completely - who knows what future log options will be needed for ARM?